pyro/poutine/trace_struct.py

```diff
@@ -78,9 +78,11 @@ def __init__(self, graph_type="flat"):
         self._succ = OrderedDict()
         self._pred = OrderedDict()
 
+    def __contains__(self, name):
+        return name in self.nodes
```

Comment:
Can't believe we were iterating through all the nodes. 😄 Missing this was probably my handiwork while refactoring to remove `networkx`.